src/lambda/lambdaTreeDataProvider.ts

```diff
@@ -9,21 +9,15 @@ import { AwsContextTreeCollection } from '../shared/awsContextTreeCollection'
 import { ext } from '../shared/extensionGlobals'
 import { RegionProvider } from '../shared/regions/regionProvider'
 import { ResourceFetcher } from '../shared/resourceFetcher'
-import { Datum, TelemetryNamespace } from '../shared/telemetry/telemetryTypes'
-import { defaultMetricDatum, registerCommand } from '../shared/telemetry/telemetryUtils'
+import { TelemetryNamespace } from '../shared/telemetry/telemetryTypes'
+import { registerCommand } from '../shared/telemetry/telemetryUtils'
 import { AWSCommandTreeNode } from '../shared/treeview/awsCommandTreeNode'
 import { AWSTreeNodeBase } from '../shared/treeview/awsTreeNodeBase'
 import { RefreshableAwsTreeProvider } from '../shared/treeview/awsTreeProvider'
 import { intersection, toMap, updateInPlace } from '../shared/utilities/collectionUtils'
-import { ChannelLogger, localize } from '../shared/utilities/vsCodeUtils'
-import {
-    applyResultsToMetadata,
-    createNewSamApplication,
-    CreateNewSamApplicationResults
-} from './commands/createNewSamApp'
+import { localize } from '../shared/utilities/vsCodeUtils'
```

Comment:
Remove this import and use nls directly like the rest of the codebase.

Reply:
I'm all for this, but it isn't within the scope of this PR. This line is simply removing `ChannelLogger` from:
`import { ChannelLogger, localize } from '../shared/utilities/vsCodeUtils'`

Reply:
Sounds good, let's fast-follow if we can while it's on our minds :)